### PR TITLE
easier to understand messages / prompts for user

### PR DESCRIPTION
**Pull request by gord5500**
### Why

Making it easier to update prompt text and not showing exception messages users likely won't understand

Swapped the order of commands and options for most useful to likely least

Chenfeng had an existing branch for this and I have just fixed the merge issues

### CODE REVIEW

File: src/D2L.Bmx/Program.cs [hunk 1]
```diff
@@ -106,32 +134,30 @@
...
-var writeCommand = new Command( "write", "Write to AWS credentials file" ) {
+var writeCommand = new Command( "write", "Write AWS credentials to the credentials file" ) {
 	accountOption,
+	roleOption,
 	durationOption,
 	orgOption,
 	outputOption,
 	profileOption,
-	roleOption,
 	userOption,
```
Comment (cfbao) [inline src/D2L.Bmx/Program.cs:156]: The order here looks off.
Profile should be higher up. Org should be together with user.
I'd say: profile, duration, output, org, user

Comment (gord5500) [reply src/D2L.Bmx/Program.cs:156]: Hmm yea it is off compared to the others where org and user are close. I've adjusted to your suggested order

**Comment by gord5500** [comment]
Maybe I got affected by this because my commit from yesterday didn't show up until I made an empty commit and pushed

https://www.githubstatus.com/incidents/pr0ptqcw6q8d

File: src/D2L.Bmx/Program.cs [hunk 2]
```diff
@@ -106,32 +134,30 @@
...
+	description: ParameterDescriptions.Profile );
 
-var writeCommand = new Command( "write", "Write to AWS credentials file" ) {
+var writeCommand = new Command( "write", "Write AWS credentials to the credentials file" ) {
 	accountOption,
+	roleOption,
 	durationOption,
-	orgOption,
-	outputOption,
 	profileOption,
```
[the remark this answers is not in the document]
Comment (gord5500) [reply src/D2L.Bmx/Program.cs:152]: No worries I've swapped them back 😃

**Review by cfbao — APPROVED** [review]

**Pull request merged**